### PR TITLE
Set Chart to full width

### PR DESCRIPTION
**Pull request by JasonMHasperhoven**
Fixes the issue that the chart doesnt span the full width:

image

### CODE REVIEW

**Comment by grod220** [comment]
Can you provide a screenshot of the change and a description on the PR? 🙏

**Comment by JasonMHasperhoven** [comment]
> Can you provide a screenshot of the change and a description on the PR? 🙏

Apologies, added 👍

File: src/components/charts/ohlcChart.tsx [hunk 1]
```diff
@@ -2,31 +2,32 @@
...
   const [isOHLCDataLoading, setIsOHLCDataLoading] = useState(true);
   const [isTimestampsLoading, setIsTimestampsLoading] = useState(true);
   const [ohlcData, setOHLCData] = useState([]); // [{open, high, low, close, directVolume, swapVolume, height}]
   const [originalOHLCData, setOriginalOHLCData] = useState([]); // [{open, high, low, close, directVolume, swapVolume, height}
   const [blockToTimestamp, setBlockToTimestamp] = useState<Record<string, string>>({}); // {height: timestamp}
   const [error, setError] = useState<string | undefined>(undefined); // [error message]
-  const [chartData, setChartData] = useState<
-    [string, number, number, number, number][]
-  >([]); // [[date, open, close, low, high]]
+  const [chartData, setChartData] = useState<[string, number, number, number, number][]>([]); // [[date, open, close, low, high]]
```
Comment (grod220) [inline src/components/charts/ohlcChart.tsx:27]: _(not for later)_
At some point, would be great to convert this data structure to an object versus ordered array.

File: src/components/charts/ohlcChart.tsx [hunk 2]
```diff
@@ -40,18 +41,19 @@ const OHLCChart = ({ asset1Token, asset2Token }: OHLCChartProps) => {
...
-          return startBlock
+      const startBlock = await fetch('/api/blocks/1')
+        .then(res => res.json())
+        .then(data => {
+          const currentBlock = data[0].height;
+          console.log(currentBlock);
+          const startBlock =
+            currentBlock - Math.trunc((daysLookback * 24 * 60 * 60) / blockTimeSeconds);
+          console.log('Start block: ', startBlock);
+          return startBlock;
```
Comment (grod220) [inline src/components/charts/ohlcChart.tsx:52]: _(not for later)_
Would be nice to stick with async/await or then()'s and not mix.

**Review by grod220 — APPROVED** [review]
Minor comments for later, not actionable now. LGTM.

**Review by philipjames44 — APPROVED** [review]

**Pull request merged**